addon/components/polaris-choice-list.js

```diff
@@ -99,11 +117,29 @@ export default Component.extend({
    *
    * @property onChange
    * @public
-   * @type {function}
+   * @type {Function}
    * @default noop
    */
   onChange() {},
 
+  /**
+   * @private
+   */
+  ariaInvalid: computed('error', function() {
+    return this.get('error') != null;
+  }),
+
+  /**
+   * @private
+   */
+  ariaDescribedBy: computed('finalName', 'error', function() {
+    if (isNone(this.get('error'))) {
+      return null;
+    }
```

Comment:
This check doesn't seem to be in the [original](https://github.com/Shopify/polaris/blob/master/src/components/ChoiceList/ChoiceList.tsx#L121), any reason for adding it?

Reply:
Ahh no, I think this is a combination of my brain going rogue as well as a hilarious misunderstanding of what `aria-describedby` is for 🤦‍♂️